windows/Gyroscope.cs

```diff
@@ -29,20 +30,24 @@ private void ReadingChanged(object sender, Sensors.GyrometerReadingChangedEventA
         {
             Sensors.GyrometerReading reading = e.Reading;
 
-            this.SendEvent("Gyroscope", new RNSensorsJsonObject
+            if (_lastReading.AddMilliseconds(interval) <= reading.Timestamp)
```

Comment:
This means the events come more often on the native side, but we only forward then eventually. Can we somehow configure the API to only send events in a given interval?